modules/passkey/test/utils/webauthn.ts

```diff
@@ -9,8 +9,25 @@
 
 import { p256 } from '@noble/curves/p256'
```

Comment:
Last nit, I promise :sweat_smile: - now that this only has the WebAuthn shim code, I think we should `git mv modules/passkey/test/utils/webauthn.ts modules/passkey/test/utils/webauthnShim.ts`

Reply:
Done 😅